src/Activation/ProductAnalyticsConsentForm.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { FunctionComponent } from "react"
+import {
+  Linking,
+  View,
+  StyleSheet,
+  ScrollView,
+  TouchableOpacity,
+} from "react-native"
+import { useNavigation } from "@react-navigation/native"
+import { useTranslation } from "react-i18next"
+
+import { useConfigurationContext } from "../ConfigurationContext"
+import { useProductAnalyticsContext } from "../ProductAnalytics/Context"
+import { ActivationStackScreens, useStatusBarEffect } from "../navigation"
+
+import { Text } from "../components"
```

Comment:
I think stuff from `components` may want to live in the second group of imports.